src/stacks-hierarchy/Root_LocationSearchByTextScreen/components/LocationSearchContent.tsx

```diff
@@ -224,6 +225,24 @@ export function LocationSearchContent({
   );
 }
 
+const useOnlyStopPlacesState = (): [boolean, (b: boolean) => void] => {
```

Comment:
Maybe we should have a general util for this, something like `useLocallyStoredState`?